lib/data/network/invitation/invitation_api.dart

```diff
@@ -61,4 +62,28 @@ class InvitationAPI {
 
     return GenerateInvitationLinkResponse.fromJson(response);
   }
+
+  Future<InvitationStatusResponse> getInvitationStatus({
```

Comment:
because it will call many times when scroll the contact list -> should have memory cache for this request